easybuild/easyconfigs/i/impi/impi-2018.1.163-GCC-6.4.0-2.28.eb

```diff
@@ -0,0 +1,36 @@
+# This is an easyconfig file for EasyBuild, see http://easybuilders.github.io/easybuild
+
+name = 'impi'
+version = '2018.1.163'
+
+homepage = 'http://software.intel.com/en-us/intel-mpi-library/'
+description = "Intel MPI Library, compatible with MPICH ABI"
+
+toolchain = {'name': 'GCC', 'version': '6.4.0-2.28'}
+
+sources = ['l_mpi_%(version)s.tgz']
+
+checksums = ['130b11571c3f71af00a722fa8641db5a1552ac343d770a8304216d8f5d00e75c']
+
+dontcreateinstalldir = 'True'
+
+components = ['intel-mpi', 'intel-psxe', 'intel-imb']
+
+# set up all the mpi commands to default to intel compilers
+# set_mpi_wrappers_all = 'True'
+
+postinstallcmds = [
+    'ln -s %(installdir)s/lib64/libmpi.so  %(installdir)s/lib64/libmpich.so',
+    'ln -s %(installdir)s/lib64/libmpigc4.so  %(installdir)s/lib64/libmpichcxx.so',
+    'ln -s %(installdir)s/lib64/libmpigf.so %(installdir)s/lib64/libfmpich.so',
+    'ln -s %(installdir)s/lib64/libmpigf.so %(installdir)s/lib64/libmpichf90.so',
+    'ln -s %(installdir)s/lib64/libmpi.so %(installdir)s/lib64/libmpl.so',
+    'ln -s %(installdir)s/lib64/libmpi.so %(installdir)s/lib64/libopa.so',
+    "perl -pi -e 's;(^#!/bin/sh);$1\nunset I_MPI_PMI_LIBRARY;' %(installdir)s/intel64/bin/mpirun",
+]
+
+modextravars = {
+    'I_MPI_PMI_LIBRARY': '/lap/slurm/lib/libpmi.so',
```

Comment:
This is a site-specific customisation, can we make this optional through the `impi` easyblock?

Reply:
Oops, sorry, I knew there was one more thing i needed to move into the hook

Reply:
And yes, the same comment applies to both of them.